cmd/influx/cli/cli.go

```diff
@@ -66,9 +67,11 @@ func New(version string) *CommandLine {
 }
 
 // Run executes the CLI
-func (c *CommandLine) Run() {
+func (c *CommandLine) Run() int {
```

Comment:
Personally I'd rather see `Run()` return an `error` and the caller can write it out to `os.Stderr`. That would make it easier to evaluate which errors are returned from the tests.